assets/javascripts/helpers/ifCond.js.es6

```diff
@@ -0,0 +1,17 @@
+//This helper class is performing a greater than operation on given two values and returning True or False
+// based on the evaluation
+import { registerHelper } from 'discourse-common/lib/helpers';
+
+
+registerHelper('ifCond', function(params) {
```

Comment:
Since you're only using this helper in one place, it might make more sense to use a more specific one to your purpose. Additionally, you could create a computed property in a component that does the check (that's usually how we do it in discourse.)

Reply:
Re-jiggered and renamed the helper to be more specific.